static/app/views/starfish/components/spanDescriptionLink.tsx

```diff
@@ -32,6 +37,7 @@ export function SpanDescriptionLink({
     project: projectId,
     endpoint,
     endpointMethod,
+    ...(spanOp ? {[SPAN_OP]: spanOp} : {}),
```

Comment:
When you click on a image resource, we'll make sure to add the resource span op as a query param here.